SimG4CMS/Forward/src/MtdSD.cc

```diff
@@ -40,6 +41,11 @@ MtdSD::MtdSD(const std::string& name,
   if (scheme)
     setNumberingScheme(scheme);
 
+  energyCut = m_p.getParameter<double>("EnergyThresholdForPersistencyInGeV") * CLHEP::GeV;     //default must be 0.5
+  energyHistoryCut = m_p.getParameter<double>("EnergyThresholdForHistoryInGeV") * CLHEP::GeV;  //default must be 0.05
+
+  setCuts(energyCut, energyHistoryCut);
```

Comment:
this statement would treat tracks in BTL and in ETL in the same way, which is probably something we do not want. I suggest we possibly discuss this for a further development.

Reply:
@fabiocos , yes, in the next rounds I would prefer remove these lines from all SD classes - it is not their job. For now this PR is fine. @srimanob, would you agree?